components/CreateForm/Price.tsx

```diff
@@ -27,6 +27,9 @@ export default function Price() {
               setPrice(val);
             }
           }}
+          onWheel={(e) => {
+            e.currentTarget.blur();
+          }}
```

Comment:
would the same functionality exist with less code if we just removed the `step` prop on line 22?